packages/backend/src/apps/telegram-bot/dynamic-data/get-chat-ids.ts

```diff
@@ -35,15 +45,18 @@ export default defineDynamicData({
   name: 'Get Telegram Chat IDs',
   async run($: IGlobalVariable): Promise<DynamicDataOutput> {
     const chatIdsMap: { name: string; value: string }[] = []
-    const chatIdsSet = new Set<string>()
+    const chatIdsSet = new Set<number>()
     try {
-      const { data } = await $.http.get(getUpdatesApi)
+      const { data }: { data: TelegramGetUpdatesResponse } = await $.http.get(
```

Comment:
nit: can use generics 
```suggestion
      const { data } = await $.http.get<TelegramGetUpdatesResponse>(
```

Reply:
Thanks!